### PR TITLE
[Kotlin Cleanup] Various

### PR DESCRIPTION
**Pull request by oyeraghib**
## Purpose / Description
[Kotlin Cleanup] Various Files

## Checklist

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt [hunk 1]
```diff
@@ -113,17 +113,16 @@ import kotlin.math.min
 
 @Suppress("LeakingThis") // The class is only 'open' due to testing
 @KotlinCleanup("scan through this class and add attributes - not started")
```
Comment (oyeraghib) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt:115]: Not sure what exactly this Cleanup about

File: AnkiDroid/src/test/java/com/ichi2/anki/ActivityStartupUnderBackupTest.kt [hunk 2]
```diff
@@ -38,9 +38,7 @@ import java.util.stream.Collectors
 
 @RunWith(ParameterizedRobolectricTestRunner::class)
 @Config(application = EmptyApplication::class) // no point in Application init if we don't use it
-@KotlinCleanup("IDE lint")
 @KotlinCleanup("See if we can combine Parameter and JvmField")
```
Comment (oyeraghib) [inline AnkiDroid/src/test/java/com/ichi2/anki/ActivityStartupUnderBackupTest.kt:41]: Not sure what to do here as well.

**Review by Arthur-Milchior — COMMENTED** [review]
There are far too many things here. It makes it harder to review than necessary. Changing the attribute name could be a PR in itself. Would be easy to merge quickly, and ensure that if I request you to make change, I will only have to re-review those changes.
Same with replacing `is` by `equalTo`, can be done separately.

As a consequence, if there are part where you don't know what to do, it's not an issue at all. Either you can ignore them and work on what you know. Or come on our discord, or in the pull request where the line was added, to request more detailled explanation.

**Comment by Arthur-Milchior** [comment]
Thanks for taking some of my remarks into account. However, I admit I'm confused, since you still have a lot of unrelated changes in a single commit. 
If my request was not clear, don't hesitate to ask for clarification. You can talk on discord too, this is standard here to ask for atomic commits and meaningful commit messages, plenty of people can probably help you.
I'd really appreciate if you could either split commits into small parts, or maybe just do multiple PRs out of it so that they can be reviewed and merged easily with certainty to have it right

**Comment by oyeraghib** [comment]
This PR has been converted to atomic PR's, so I am closing it.

**Pull request closed without merging**